gdal/doc/source/development/rfc/rfc80_numfocus_relationship.rst

```diff
@@ -0,0 +1,135 @@
+
+A `Pre-approved Grantor/ Grantee Agreement <https://docs.google.com/document/d/12KGmSD_EGWqlzh0fiaHNIiN6xPWgdTy--NYf4QR-5k8/edit>`__
+will also have to be signed by 3 members of the GDAL PSC.
+
+General use of funds
+--------------------
+
+This topic will be refined in a further document, but we will underline here
+the general principles.
+
+Funds obtained through NumFOCUS should be used in a way that is for the general good
+of the project, as well as its key open source dependencies (such as PROJ, libgeotiff,
+libtiff, shapelib, etc.).
+
+The tasks candidate to funding will be along (non-exhaustive
+list):
+
+- ticket triaging and fixing,
+- code contribution review,
+- continuous integration maintenance and improvements,
+- monitoring of mailing list,
+- addressing reports raised by security monitoring infrastructure,
+- documentation improvement,
+- software releases,
+- co-operation with other funded developers (in particular onboarding),
+- reports of those activities
+
+More generally, all tasks that are needed for a project to strive on the long term, but are
+typically hard to get directly funding from, as not being directly tied to a
+particular feature. It is expected that new features and
+improvements will go on being submitted as code contributions by the interested
+parties in capacity to develop them, or through service providers they might hire
+for that purpose.
+
+Sponsors will not be able to direct particular developments, nor receive priority
+support through the sponsorship program. For such purposes, they may use
+community support or refer to service providers, as for other GDAL users.
+However, we will offer a way to sponsors to give feedback, as detailed afterwards.
+
+GDAL Advisory Board
+-------------------
+
+The GDAL Advisory Board is a newly forming group that is tasked with ensuring the project is evolving to meet the needs of its largest supporters. There will be periodic meetings with the advisory board, GDAL PSC members wishing to participate, and persons benefiting from funding, providing a forum to give feedback on the latest GDAL work. Board members will also be expected to share how they are using GDAL in their organization (in as much detail as is possible, understanding that some can only talk in general terms). Though the GDAL PSC owns the decision making around how the sponsorship money is allocated, the advisory board will provide key input on the maintenance priorities of the sponsors.
+Our hope is that these sessions can also lead to new feature development in GDAL through joint funding initiatives between top sponsors, providing more structure to the way GDAL features are funded today. This ideally leads to more cost-sharing as organizations will be able to be more strategic when they know what everyone else is interested in. These efforts will be above and beyond the core maintenance sponsorship, but the Advisory Board will be a key forum to enable it.
```

Comment:
Formatting nit here.  Add a blank line rather than a leading space.

Reply:
there was no leading space AFAICS (probably just the artifact of the "+" of the diff), but blank line added for readability